tracing/src/main/java/com/palantir/tracing/Tracer.java

```diff
@@ -331,8 +331,18 @@ public static boolean isTraceObservable() {
     static void setTrace(Trace trace) {
         currentTrace.set(trace);
 
-        // Give SLF4J appenders access to the trace id
+        // Give SLF4J appenders access to the trace id and if trace is being sampled
```

Comment:
> SLF4J appenders

No such thing!

> and if trace is being sampled

Maybe `s/if/whether`, the first couple reads through this comment I thought there was a piece missing from the end.